src/test/resources/fixed_demographics/fixed_demographics_test.json

```diff
@@ -0,0 +1,274 @@
+[
+  {
+    "records": [
+      {
+        "LIST_ID": "DH",
+        "RECORD_ID": "19436338",
+        "CHILD_SURNAME": "Fredrich",
+        "CHILD_GIVEN_NAME": "Alex / Alex Neil",
+        "DOB_YEAR": "2014",
+        "DOB_MONTH": "7",
+        "DOB_DAY": "30",
+        "GENDER": "M",
+        "PHONE_AREA_CODE": "303",
+        "PHONE_NUMBER": "9373705",
+        "ADDRESS_STREET1": "12439 Josephine St",
+        "ADDRESS_STREET2": "",
+        "ADDRESS_CITY": "Thornton",
+        "ADDRESS_STATE": "Colorado",
+        "ADDRESS_COUNTRY": "",
+        "ADDRESS_ZIPCODE": "80241-2744",
+        "PARENT1_SURNAME": "Fredric",
+        "PARENT1_GIVEN_NAME": "Kimberly Adrienne",
+        "PARENT1_EMAIL": "honeyanneristm@gmail.com"
```

Comment:
I am extremely curious about where the email addresses in this file came from, especially this one, but more importantly we shouldn't include any real email addresses (or any other real individual location or contact information for that matter) in the repo.  Can you change these to either an invalid domain or `@example.com` or `@example.org` , and confirm the other addresses and phone numbers don't point to something real?

Reply:
I believe all of the information in the file is completely fake, but they are from @eedrummer so he can confirm. I will also go ahead and update the domains to `@example.com`.

Reply:
I've updated all the phone numbers/emails in the test files to be clearly synthetic with repeating number patterns like `343434@example.com`.